Sequence Diagram for Customer and Service Provider

rldevries opened the pull request:
This could be broken down into 2 separate diagrams, but the intent is to provide an overview of both 'Service Requestor and Service Provider' to engage with the same website - thoughts?

schurch82 commented:
I think it should be split into two as we have to provide more sequence diagrams as per the documentation instructions, otherwise good.  There is a merge confliction with the documentation.docx file. not sure why. we will have to sort that out before merging and also before we do any further work on the documentation.docx file

rldevries commented:
Deleted original sequence_dia and replaced with SequenceServiceProvider.drawio creating 1 of 3 Sequence Diagrams.  Remaining 2 are:
- SequenceCustomer.drawio
- SequenceAdmin.drawio
The conflicts can be easily resolved through retention of only the SequenceServiceProvider.drawio, all other conflicts should keep main's.
